babybuddy/middleware.py

```diff
@@ -49,7 +78,8 @@ def __call__(self, request):
         user = request.user
         if hasattr(user, 'settings') and user.settings.language == 'en-US':
             update_en_us_date_formats()
-
+        elif hasattr(user, 'settings') and user.settings.language == 'en-GB':
```

Comment:
@Amith211 instead of doing the same check twice here, move both of the `user.settings.language` checks under a single conditional for `hasattr(user, 'settings')`.